src/elements/content-sidebar/activity-feed/comment/__tests__/RepliesToggle.test.js

```diff
@@ -0,0 +1,91 @@
+// @flow
+
+import React from 'react';
+import { IntlProvider } from 'react-intl';
+import { fireEvent, render, screen } from '@testing-library/react';
+import RepliesToggle from '../RepliesToggle';
+
+jest.mock('react-intl', () => ({
+    ...jest.requireActual('react-intl'),
+}));
+
+const showReplies = jest.fn();
+const hideReplies = jest.fn();
+const shownCount = 1;
+const totalCount = 9;
+
+const getWrapper = props =>
+    render(
+        <IntlProvider locale="en">
+            <RepliesToggle
+                id="123"
+                isRepliesLoading={false}
+                onShowReplies={showReplies}
+                onHideReplies={hideReplies}
+                repliesShownCount={shownCount}
+                repliesTotalCount={totalCount}
+                {...props}
+            />
+        </IntlProvider>,
+    );
+
+describe('elements/content-sidebar/ActivityFeed/comment/RepliesToggle', () => {
+    test('should correctly render show toggle', () => {
+        const countDifference = totalCount - 1;
```

Comment:
The value of this variable is confusing me a bit. Is the `- 1` because `shownCount` is initially 1? Or is it an index offset or something different?

Reply:
Yeah it's because the shownCount is initially 1, so when creating the message it would use the difference: `See ${difference} replies`